asserts/toolbelt/main.go

```diff
@@ -38,9 +38,20 @@ var parser = flags.NewParser(nil, flags.Default)
 var db *asserts.Database
 
 func main() {
-	var err error
+	topDir := "snappy-asserts-toolbelt-db"
+	bs, err := asserts.OpenFilesystemBackstore(topDir)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	keypairMgr, err := asserts.OpenFilesystemKeypairManager(topDir)
```

Comment:
I'd be happy to be consistent everywhere, since this a very common abbreviation. For now, can we be consistent inside asserts at least? I'd be happy to avoid `OpenFilesystemKeypairManager` as a thing.